threescale_api/auth.py

```diff
@@ -4,33 +4,35 @@
 
 import requests
 
+
 class BaseClientAuth(requests.auth.AuthBase):
     "Abstract class for authentication of api client"
     def __init__(self, app, location):
         self.app = app
-        self.location = location
+        self.location = app.service.proxy.list().entity["credentials_location"]
 
     @property
     @abc.abstractmethod
     def credentials(self):
         "returns credentials pair as tuple"
 
-    def __call__(self, r):
+    def __call__(self, request):
```

Comment:
this renaming doesn't seem to be relevant to the change in this PR, rather shouldn't be in this PR

Reply:
Resolved based on our conversation